__tests__/app.test.js

```diff
@@ -41,4 +41,37 @@ describe('/api',()=>{
             expect(endpoints).toHaveProperty('GET /api/articles')
         })
     })
+})
+describe('/api/articles/:article_id', () => {
+    test('GET:200 sends a single article to the client', () => {
+      return request(app)
+        .get('/api/articles/1')
+        .expect(200)
+        .then(({body : {article}}) => {
+            expect(article.author).toBe("butter_bridge")
+            expect(article.title).toBe("Living in the shadow of a great man");
+            expect(article.article_id).toBe(1);
+            expect(article.body).toBe("I find this existence challenging",)
+            expect(article.topic).toBe('mitch');
+            expect(article.created_at).toBe('2020-07-09T20:11:00.000Z');
+            expect(article.votes).toBe(100);
+            expect(article.article_img_url).toBe("https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700")
+        });
+    });
+    test('GET:404 sends an appropriate status and error message when given a valid but non-existent id', () => {
+      return request(app)
+        .get('/api/articles/999')
+        .expect(404)
+        .then(({body : {message}}) => {
+          expect(message).toBe('article does not exist');
+        });
+    });
+    test('GET:400 sends an appropriate status and error message when given an invalid id', () => {
+      return request(app)
+        .get('/api/articles/not-an-article-id')
+        .expect(400)
+        .then(({body: {message}}) => {
+          expect(message).toBe('Bad request');
+        });
+    });
```

Comment:
Great testing! You have covered all the bases here well done. Nice destructuring too!